mirrord/layer/src/error.rs

```diff
@@ -242,6 +242,7 @@ impl From<HookError> for i64 {
                 // this could be changed by waiting for the Subscribed response from agent.
                 ResponseError::PortAlreadyStolen(_port) => libc::EINVAL,
                 ResponseError::NotImplemented => libc::EINVAL,
+                ResponseError::Forbidden { .. } => libc::EACCES,
```

Comment:
imo we should kill the app when this happens, since it's not a "regular flow" - like the app code can try to handle this exception which will just lead to bad UX.

Reply:
Not sure what that means for this line, but I'll make sure to make the process terminate on blocked operations. Maybe server (operator) side. Maybe both.

Reply:
This line translates the forbidden error into libc errno as part of propagating it to the user app, which should be unreachable if we terminate on that message.

Reply:
So what should be here instead? We still need that match arm. Should it be `unreachable!()`? I thought we didn't like panicking (`unwrap` etc.).

Reply:
Either unreachable or call exit with the exit handler (show message etc)